config/webpack/loaders.js

```diff
@@ -5,6 +5,14 @@
 
 'use strict';
 
+var es6Loader;
+
+if (process.env.JASMINE) {
+    es6Loader = 'babel?plugins[]=rewire&plugins[]=object-assign';
```

Comment:
I tried the approach you suggested first and it though everything after the comma were options for the rewire plugin. The `[]` approach is a standard HTTP query string approach for arrays so I thought I'd give it a shot since I couldn't find anything online as to how to do this. That approach worked as I tested this with an app in which uses `Object.assign`.